lib/dpul_collections/indexing_pipeline.ex

```diff
@@ -238,6 +238,25 @@ defmodule DpulCollections.IndexingPipeline do
     |> FiggyRepo.all()
   end
 
+  def get_figgy_parents(id) do
```

Comment:
Looks like https://github.com/pulibrary/dpul-collections/pull/391 got lost in a rebase?